router/pump.go

```diff
@@ -89,6 +89,15 @@ func (p *LogsPump) Setup() error {
 	return nil
 }
 
+func (p *LogsPump) rename(event *docker.APIEvents) {
+    p.mu.Lock()
+    defer p.mu.Unlock()
+    container, err := p.client.InspectContainer(event.ID)
```

Comment:
Seems that `event.ID` has been removed in 1.22. Although [go-dockerclient](https://github.com/fsouza/go-dockerclient/blob/master/event.go#L39-L42) is maintaining backward compatability, I think it would be best to use `event.Actor.ID` in new code. What do you think?

https://github.com/fsouza/go-dockerclient/blob/master/event.go#L39-L42